internal/version/version.go

```diff
@@ -19,5 +19,5 @@ func BaseVersion() string {
 		return BuildDate
 	}
 
-	return fmt.Sprintf("%d.%d", v.Major(), v.Minor())
+	return fmt.Sprintf("v%d.%d", v.Major(), v.Minor())
```

Comment:
Prefix the letter 'v' to prevent YAML parsing from converting a '1.0' string into a number, i.e. 0.0 to 0, 1.0 to 1, etc.